Essentials/src/com/earth2me/essentials/commands/Commandclearinventory.java

```diff
@@ -79,11 +80,28 @@ private void parseCommand(Server server, CommandSource sender, String commandLab
         }
     }
 
-    protected void clearHandler(CommandSource sender, Player player, String[] args, int offset, boolean showExtended) throws Exception {
-        short data = -1;
+    private static class Item {
+        private Material material;
+        private short data;
+
+        public Item(Material material, short data) {
+            this.material = material;
+            this.data = data;
+        }
+
+        public Material getMaterial() {
+            return material;
+        }
+
+        public short getData() {
+            return data;
+        }
+    }
+
+    protected void clearHandler(CommandSource sender, Player player, String[] args, int offset, boolean showExtended) {
         int type = -1;
```

Comment:
I'd prefer this was replaced with a private enum, as done in #3191 - this variable only exists to control flow later on, and it's clearer if it uses a named enum constant than a magic number.